lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -474,7 +475,7 @@ module.exports = class MethodMappings {
           const element = this.getWebElement(webElement);
 
           // eslint-disable-next-line
-          const fn = function(e,a,v){try {if(e&&typeof e.setAttribute=='function'){e.setAttribute(a,v)} return true} catch(err){return {error:err.message,message:err.name+': '+err.message}}};
+          const fn = Utils.setAttributeFn;
```

Comment:
This is only used as an argument to execute() to be ran in the browser context. Moving it to Utils doesn't add any benefit as it cannot be used in the Node context.

Reply:
While running tests with coverage, this causes test failures as the code being executed is instrumented with coverage tracking code while, the string being tested against is not: 

https://github.com/nightwatchjs/nightwatch/pull/3039/files#diff-3a747a55f06a0eabc7bfbe77f5008eb5b566f146d930ee8533bcc31ce0df1353L16

Reply:
Then you should make it a string instead, which is going to become anyway.

Reply:
okay